docs/advanced-features/codemods.md

```diff
@@ -17,6 +17,44 @@ Codemods are transformations that run on your codebase programmatically. This al
 - `--dry` Do a dry-run, no code will be edited
 - `--print` Prints the changed output for comparison
 
+## Next.js 13
```

Comment:
@ijjk @timneutkens Do you know if the sections on [this page](https://nextjs.org/docs/advanced-features/codemods) are supposed to represent the version you are upgrading "from" or "to"? Or is it the version that introduce the code mod? 🤔

Reply:
It's "to" so you added it correctly 👍